src/cmd-koji-upload

```diff
@@ -555,7 +570,11 @@ class Reserve(_KojiBase):
             'token': koji_reservation['token'],
             'release': build.release
         }
-        build.meta_write()
+        if hasattr(build, 'meta_file_path') and build.meta_file_path:
+             with open(build.meta_file_path, "w") as f:
+                 json.dump(build.meta, f, indent=2)
```

Comment:
Hmm, this is overwriting the `meta.json` we extracted from the node image IIUC?

Since we're not actually uploading this info anywhere, it seems pointless to write it out. I guess we can instead just... print it on stdout maybe?

Reply:
Yes, it is  adding the koji rersation in met.json and uploading it as well to brew, as we do today, it was also a request in the past from the prodSec, I don't recall why, I will need to look, maybe we upload it to aws and with the id is a way to find it in brew?

Reply:
Here is the info added to meta.json that goes to brew as well:
"koji": {
        "build_id": 123,
        "token": "adfa123",
        "release": "0"
    }

Reply:
Ahh gotcha. I missed that this final version is what gets uploaded to Brew.